src/pages/ReportParticipantsPage.js

```diff
@@ -100,7 +100,8 @@ function ReportParticipantsPage(props) {
                 <FullPageNotFoundView shouldShow={_.isEmpty(props.report) || ReportUtils.isArchivedRoom(props.report)}>
                     <HeaderWithBackButton
                         title={props.translate(
-                            ReportUtils.isChatRoom(props.report) ||
+                            ReportUtils.isGroupChat(props.report) ||
```

Comment:
Indentation is wrong here?

Reply:
I don't get it @MonilBhavsar my VS Code auto prettifies according to the project config.